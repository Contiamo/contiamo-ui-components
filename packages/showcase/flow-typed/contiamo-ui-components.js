declare module "contiamo-ui-components" {
  declare var exports: any
}
