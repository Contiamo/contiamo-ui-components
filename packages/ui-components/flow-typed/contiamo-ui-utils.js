declare module "contiamo-ui-utils" {
  declare var exports: any
}
