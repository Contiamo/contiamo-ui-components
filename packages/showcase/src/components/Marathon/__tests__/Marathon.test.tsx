import * as React from "react"
import { render } from "enzyme"
import { wrapTheme } from "contiamo-ui-utils"
import { contiamoTheme } from "contiamo-ui-components"
import ThemelessMarathon from "../Marathon"

const Marathon = wrapTheme(contiamoTheme)(ThemelessMarathon)

describe("Marathon Component", () => {
  it("Should render", () => {
    const renderedComponent = render(<Marathon />)
    expect(renderedComponent).toMatchSnapshot()
  })
})
