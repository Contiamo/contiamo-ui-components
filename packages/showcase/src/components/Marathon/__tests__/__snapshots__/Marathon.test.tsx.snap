// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Marathon Component Should render 1`] = `
<div
  class="css-nil"
>
  <ul
    class="css-1a24qct"
  />
  <div
    class="css-1dd5b0s"
  />
</div>
`;
